src/wwwroot/services/settings.js

```diff
@@ -161,5 +163,18 @@
       });
     }
 
+    function getCurrentPlanInfo() {
```

Comment:
I am not sure if it is a good idea to cache this result. What do you think @mvillaseco ?